### DISCUSSION TITLE
https://wj-mcat.github.io/agent-handbook/docs/agent-intro/what-is-agent

### DISCUSSION THREAD

**Discussion by giscus** [wj-Mcat/agent-handbook#4] [General]
# https://wj-mcat.github.io/agent-handbook/docs/agent-intro/what-is-agent

相信大家对 Agent 的概念并不陌生，已然有无数软文介绍过，在此我简要介绍一下：Agent 是一个能够调用外部工具、拥有长期记忆以及根据已有信息做有效任务规划的智能体。

https://wj-mcat.github.io/agent-handbook/docs/agent-intro/what-is-agent

**Comment by wj-Mcat** [comment]
感觉文字还是稍显简单

**Reply by wj-Mcat** [reply]
test
